app/controllers/users_controller.rb

```diff
@@ -96,9 +96,13 @@ def impersonate
   def invalidate_jwt
     @user = find_resource(:edit_users)
     @user.jwt_secret&.destroy
-    process_success(
-      :success_msg => _('Successfully invalidated JWTs for %s.') % @user.login
-    )
+    unless editing_self?
```

Comment:
Why this change? Why not to have same response as before?

Reply:
@girijaasoni I think we should use `respond_to` here. It would be more consistent.